### ISSUE TITLE
Is this safe to use in competitive?

### ISSUE THREAD

**Issue by Jakedanice** [Jire/Charlatano#610]
Is this safe to use in competitive?

**Comment by Jire** [comment]
Yes, but exercise caution to not get yourself overwatched

**Comment by Jakedanice** [comment]
@Jire Well thats hard :p but what about wingman i just cheated there and they knew i hacked

**Comment by auserthatistrash** [comment]
yes they have OW for wingman. Just only get max 10 reports per 48 Hours
